docs/_sass/code.scss

```diff
@@ -71,7 +71,7 @@ pre {
     word-wrap: normal;
     background-color: var(--color-primary-medium);
     padding: 1rem;
-    overflow: scroll;
+    overflow: hidden;
```

Comment:
How are you guaranteeing that content is being cutoff, @thisiskaransgit?

Reply:
| Before   |      After      |
|----------|:-------------:|
| <img src="https://github.com/meshery/meshery/assets/92504666/a8fc135f-211f-41d6-a7dd-e49c577b004e" style="width:50%;"> |  <img src="https://github.com/meshery/meshery/assets/92504666/6409811e-cffa-4c96-a2eb-a218d1466df4" style="width:50%;"> |
This won't have side effects in other components

Reply:
This seems good